localization/pose_initializer/launch/pose_initializer.launch.xml

```diff
@@ -1,13 +1,14 @@
 <launch>
   <arg name="common_config_file" default="$(find-pkg-share pose_initializer)/config/pose_initializer_common.param.yaml"/>
   <arg name="config_file" default="$(find-pkg-share pose_initializer)/config/pose_initializer.param.yaml"/>
+  <arg name="gnss_pose_cov"/>
```

Comment:
Maybe it's better to follow the naming convention discussed [here](https://github.com/orgs/autowarefoundation/discussions/3171#discussioncomment-4597052).

Also, are there any reason why you don't specify a default value for the argument?

```suggestion
  <arg name="sub_gnss_pose_cov"/>
```

Reply:
@mitsudome-r 
No reason. Default value added.